owrx/feature.py

```diff
@@ -694,3 +695,11 @@ def has_paho_mqtt(self):
             return True
         except ImportError:
             return False
+
+    def has_nrsc5(self):
+        """
+        OpenWebRX uses the [nrsc5](https://github.com/theori-io/nrsc5) tool to decode HDRadio
+        FM broadcasts. Nrsc5 is not yet available as a package and thus you will have
+        to compile it from source.
+        """
+        return self.command_is_runnable("nrsc5 -v")
```

Comment:
Given that the code only depends on the library (not the actual executable), I think that it would be a good idea to change this to test for the library as well. Not sure if it will bring a big advantage, but this would allow the library to come as a separate package, reducing the dependencies when possible.

Reply:
Yeah, that should not be a problem. The original integration did use the executable, hence the test.